### PR TITLE
[stable/falco] Optional daemonset environment variables (#15762)

### LINKED ISSUE #15762: [stable/falco] Daemonset environment variables
Would like to pass in environment variables to the daemonset pods.

**Is your feature request related to a problem? Please describe.**

This requirement has come up to solve this issue, which is solved by passing an environment variable - https://github.com/falcosecurity/falco/issues/650#issuecomment-501541258

**Describe the solution you'd like**

Pass a map of environment variables.

### PR DESCRIPTION
**Pull request by billyshambrook**
Signed-off-by: Billy Shambrook <billy.shambrook@gmail.com>

#### Which issue this PR fixes
  - fixes #15762

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/falco/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: falco
-version: 1.0.0
+version: 1.1.0
```
Comment (nestorsalceda) [inline stable/falco/Chart.yaml]: Use 1.0.1 instead of 1.1.0

Thank you!

File: stable/falco/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,12 @@
 This file documents all notable changes to Sysdig Falco Helm Chart. The release
 numbering uses [semantic versioning](http://semver.org).
 
+## v1.1.0
```
Comment (nestorsalceda) [inline stable/falco/CHANGELOG.md]: Use 1.0.1 instead of 1.1.0

Thanks!

**Review by nestorsalceda — CHANGES_REQUESTED** [review]
Just a little version fix and I will approve the PR.

Thank you so much for the PR :)

File: stable/falco/Chart.yaml (hunk 1 above)
Comment (billyshambrook) [reply stable/falco/Chart.yaml]: Done

File: stable/falco/CHANGELOG.md (hunk 2 above)
Comment (billyshambrook) [reply stable/falco/CHANGELOG.md]: Yeah sure, have pushed the change, thanks!

**Comment by billyshambrook** [comment]
@nestorsalceda Thanks for reviewing, have adjusted the version as asked, thanks!!

**Review by nestorsalceda — APPROVED** [review]

**Comment by nestorsalceda** [comment]
Thank you for the PR @billyshambrook !

**Pull request merged**